### ISSUE TITLE
Hab install fails on Windows because it can't find the directory

### ISSUE THREAD

**Issue by emachnic** [chef-cookbooks/habitat#199]
### Cookbook version
1.5.1

### Chef-client version
15.0

### Platform Details
Windows 2012R2

### Scenario:
Use the `hab_install` resource

### Steps to Reproduce:
Boot up a Windows server and try to install using `hab_install`

### Expected Result:
Habitat is installed successfully

### Actual Result:
The hab-latest-x86_64-windows.zip package is downloaded successfully but when it's extracted, the version is there instead of "latest". For example in Test Kitchen, `C:\Users\Administrator\AppData\Local\Temp\kitchen\cache\habitat\hab-1.5.0-20200122222331-x86_64-windows` exists but it's trying to find `C:\Users\Administrator\AppData\Local\Temp\kitchen\cache\habitat\hab-latest-x86_64-windows`

```
* powershell_script[installing from archive] action run

             ================================================================================
             Error executing action `run` on resource 'powershell_script[installing from archive]'
             ================================================================================

             Mixlib::ShellOut::ShellCommandFailed
             ------------------------------------
             Expected process to exit with [0], but received '1'
             ---- Begin output of "C:\Windows\system32\WindowsPowerShell\v1.0\powershell.exe" -NoLogo -NonInteractive -NoProfile -ExecutionPolicy Bypass -InputFormat None -File "C:/Users/ADMINI~1/AppData/Local/Temp/chef-script20200204-1544-1enf0xv.ps1" ----
             STDOUT:
             STDERR: Cannot find path 'C:\Users\ADMINI~1\AppData\Local\Temp\kitchen\cache\habitat\hab-latest-x86_64-windows' because it does not exist.
             ---- End output of "C:\Windows\system32\WindowsPowerShell\v1.0\powershell.exe" -NoLogo -NonInteractive -NoProfile -ExecutionPolicy Bypass -InputFormat None -File "C:/Users/ADMINI~1/AppData/Local/Temp/chef-script20200204-1544-1enf0xv.ps1" ----
             Ran "C:\Windows\system32\WindowsPowerShell\v1.0\powershell.exe" -NoLogo -NonInteractive -NoProfile -ExecutionPolicy Bypass -InputFormat None -File "C:/Users/ADMINI~1/AppData/Local/Temp/chef-script20200204-1544-1enf0xv.ps1" returned 1

             Resource Declaration:
             ---------------------
             # In C:/Users/ADMINI~1/AppData/Local/Temp/kitchen/cache/cookbooks/habitat/resources/install.rb

       60:     powershell_script 'installing from archive' do
       61:       code <<-EOH
       62:       Move-Item -Path #{Chef::Config[:file_cache_path]}/habitat/#{package_name} -Destination C:/habitat -Force
       63:       EOH
       64:     end
       65:

             Compiled Resource:
             ------------------
             # Declared in C:/Users/ADMINI~1/AppData/Local/Temp/kitchen/cache/cookbooks/habitat/resources/install.rb:60:in `block in class_from_file'

             powershell_script("installing from archive") do
        action [:run]
        default_guard_interpreter :powershell_script
        command nil
        backup 5
        interpreter "powershell.exe"
        declared_type :powershell_script
        cookbook_name "uship_web_server"
        code "      Move-Item -Path C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\kitchen\\cache/habitat/hab-latest-x86_64-windows -Destination C:/habitat -Force\n"
        domain nil
        user nil
        flags "-NoLogo -NonInteractive -NoProfile -ExecutionPolicy Bypass -InputFormat None"
             end

             System Info:
             ------------
             chef_version=15.0.300
             platform=windows
             platform_version=6.3.9600
             ruby=ruby 2.6.3p62 (2019-04-16 revision 67580) [x64-mingw32]
             program_name=C:/opscode/chef/bin/chef-client
             executable=C:/opscode/chef/bin/chef-client


           ================================================================================
           Error executing action `install` on resource 'hab_install[install Habitat]'
           ================================================================================

           Mixlib::ShellOut::ShellCommandFailed
           ------------------------------------
           powershell_script[installing from archive] (C:/Users/ADMINI~1/AppData/Local/Temp/kitchen/cache/cookbooks/habitat/resources/install.rb line 60) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '1'
           ---- Begin output of "C:\Windows\system32\WindowsPowerShell\v1.0\powershell.exe" -NoLogo -NonInteractive -NoProfile -ExecutionPolicy Bypass -InputFormat None -File "C:/Users/ADMINI~1/AppData/Local/Temp/chef-script20200204-1544-1enf0xv.ps1" ----
           STDOUT:
           STDERR: Cannot find path 'C:\Users\ADMINI~1\AppData\Local\Temp\kitchen\cache\habitat\hab-latest-x86_64-windows' because it does not exist.
           ---- End output of "C:\Windows\system32\WindowsPowerShell\v1.0\powershell.exe" -NoLogo -NonInteractive -NoProfile -ExecutionPolicy Bypass -InputFormat None -File "C:/Users/ADMINI~1/AppData/Local/Temp/chef-script20200204-1544-1enf0xv.ps1" ----
           Ran "C:\Windows\system32\WindowsPowerShell\v1.0\powershell.exe" -NoLogo -NonInteractive -NoProfile -ExecutionPolicy Bypass -InputFormat None -File "C:/Users/ADMINI~1/AppData/Local/Temp/chef-script20200204-1544-1enf0xv.ps1" returned 1

           Resource Declaration:
           ---------------------
           # In C:/Users/ADMINI~1/AppData/Local/Temp/kitchen/cache/cookbooks/uship_web_server/recipes/default.rb

           177: hab_install 'install Habitat' do
           178:   license 'accept'
           179: end
           180:

           Compiled Resource:
           ------------------
           # Declared in C:/Users/ADMINI~1/AppData/Local/Temp/kitchen/cache/cookbooks/uship_web_server/recipes/default.rb:177:in `from_file'

           hab_install("install Habitat") do
             action [:install]
             updated true
             updated_by_last_action true
             default_guard_interpreter :default
             declared_type :hab_install
             cookbook_name "uship_web_server"
             recipe_name "default"
             license "accept"
           end

           System Info:
           ------------
           chef_version=15.0.300
           platform=windows
           platform_version=6.3.9600
           ruby=ruby 2.6.3p62 (2019-04-16 revision 67580) [x64-mingw32]
           program_name=C:/opscode/chef/bin/chef-client
           executable=C:/opscode/chef/bin/chef-client
```

**Comment by sirajrauff** [comment]
Had this same problem. Quick workaround, I modified `resources/install.rb:62` from:
```
Move-Item -Path #{Chef::Config[:file_cache_path]}/habitat/#{package_name} -Destination C:/habitat -Force
```
to:
```
Move-Item -Path #{Chef::Config[:file_cache_path]}/habitat/hab-1.5.0-20200122222331-x86_64-windows -Destination C:/habitat -Force
```
Seems that it's due to the way it was zipped up? Tried downloading the previous version to find that it had the same `hab-<version>-<release>-<target>` top level dir:
```
$ curl -o hab-0.90.6-x86_64-windows.zip -L https://packages.chef.io/files/stable/habitat/0.90.6/hab-x86_64-windows.zip
$ unzip -l hab-0.90.6-x86_64-windows.zip
Archive:  hab-0.90.6-x86_64-windows.zip
  Length      Date    Time    Name
---------  ---------- -----   ----
        0  11-12-2019 10:10   hab-0.90.6-20191112142421-x86_64-windows/
        ...
```
So it looks like if you download `habitat/<version>/hab-x86_64-windows.zip` you're meant to get the structure that we see above, but I'm assuming in the past if you downloaded `habitat/latest/hab-x86_64-windows.zip` you'd expect a top-level dir of `hab-latest-<target>`

**Comment by emachnic** [comment]
@sirajrauff previously, it pulled from bintray and queried the API to get the latest version number. It's now just pulling the latest off of packages.chef.io instead of a pinned version or dynamic one from API

**Comment by jonlives** [comment]
Fixed in #202
